resources/dicts/patrols/plains/training/any.json

```diff
@@ -2828,5 +2828,125 @@
                 "lost_cats": ["r_c"]
             }
         ]
+    },
+    {
+        "patrol_id": "gen_train_amentorswisdom",
+        "biome": ["Any"],
+        "season": ["Any"],
+        "types": ["training"],
+        "tags": ["platonic","trust","respect","dislike","comfort"],
+        "patrol_art": "train_general_intro",
+        "min_cats": 2,
+        "max_cats": 2,
+        "min_max_status": {
+            "normal adult": [0, 1],
+            "all apprentices": [0, 1]
+        },
+    
+        "weight": 20,
+        "chance_of_success": 50,
+        "intro_text": "p_l is surprised when app1 makes a mistake on a simple move p_l thought {PRONOUN/app1/subject} had mastered. Before p_l can say anything, app1 wails and runs out of the training grounds. When p_l finds {PRONOUN/app1/poss} huddled in camp, {PRONOUN/app1/subject} {VERB/app1/sob/sobs} out a confession: {PRONOUN/app1/subject} will never be ready to be a warrior.",
+        "decline_text": "It wouldn't help to talk to app1 while {PRONOUN/app1/subject} {VERB/app1/is/are} in this state. p_l tells {PRONOUN/app1/poss} {PRONOUN/p_l/subject}'ll leave {PRONOUN/app1/poss} alone, and to come find {PRONOUN/p_l/poss} if {PRONOUN/app1/subject}'d like to keep training today. app1 never does.",
```

Comment:
{VERB/app1/is/are} --> {VERB/app1/are/is} (the plural one comes first)

tells {PRONOUN/app1/poss} = tells their. Instead, tells {PRONOUN/app1/object} = tells them
https://docs.google.com/document/d/1qcks1E25NkOfU6zdwUlccU42VRkHv_WNaCsMpUP5jDA/edit
for pronoun help 
the poss vs object error continues in this text, please fix

I really like this decline text, I think you did a great job with it and it's a lot better than most of the ones I make lol.